core/pom.xml

```diff
@@ -102,9 +102,14 @@
             <version>3.6</version>
         </dependency>
         <dependency>
-            <groupId>com.opencsv</groupId>
-            <artifactId>opencsv</artifactId>
-            <version>4.1</version>
+            <groupId>com.univocity</groupId>
+            <artifactId>univocity-parsers</artifactId>
+            <version>2.6.3</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-text</artifactId>
+            <version>1.1</version>
```

Comment:
I'm not sure i understood your comment on the issue about commons text. Are you saying that univocity requires it, but only supports this old version, or that newer versions are somehow incompatible with tablesaw